backend/src/articles/api/views.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.generics import ListAPIView, RetrieveAPIView
+
+from articles.models import Article
+from .serializers import ArticalSerialziers
+
+
+class AricalsListView(ListAPIView):
```

Comment:
watch out for misspellings in code or have someone proofread if it's something you have difficulties with (i.e. dyslexia), if we catch them early, the less we have to refactor.

Reply:
will look out for them, ill proabbyl ask somoeone for the help, spelling really isnt my best field of work tho.